src/web/PopupContainer.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+* PopupContainer.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* Common parent of all components rendered into a popup. Calls onShow and onHide
+* callbacks when the popup is hidden (i.e. "closed" but still rendered as hidden)
+* and re-shown.
+*/
+
+import React = require('react');
+import PropTypes = require('prop-types');
+import FocusManager from './utils/FocusManager';
+
+export interface PopupContainerProps {
```

Comment:
This should extend RX.Types.CommonProps to pick up the 'children' and 'style' fields.

Reply:
Done.